changelog/646.bugfix.rst

```diff
@@ -0,0 +1 @@
+Improve channel/guild fallback in resolved interaction data.
```

Comment:
Could we explain how this was fixed and what changes this will bring to a user?

Reply:
2d755778e942cb465fcf9ba25a390448de9ed98c